### PR TITLE
Add Dhali-app

### PR DESCRIPTION
**Pull request by desimmons**
This PR adds Dhali-app as a single commit

### CODE REVIEW

**Review by robbie-daniels — COMMENTED** [review]
* Shall we remove the "Help" tab for now?  It's just a placeholder at the moment and I think it's misleading:
image

* Love the test CI!
* Shall we also clean up all of the test apps?
* Do we want to have publishing to a staging area (rather than just the temporary links we get in PRs) for merge into `develop`, and prod in `main`?  This seems desirable, since I think we want to test things outside of PRs too.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Dhali App
+
```
[the remark this answers is not in the document]
Comment (desimmons) [reply README.md:2]: I'm not sure this is true

File: firestore.indexes.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  //  "fieldOverrides": [
+  //    {
+  //      "collectionGroup": "widgets",
+  //      "fieldPath": "baz",
+  //      "indexes": [
+  //        { "order": "ASCENDING", "queryScope": "COLLECTION" }
+  //      ]
+  //    },
+  //   ]
+  // ]
```
[the remark this answers is not in the document]
Comment (desimmons) [reply firestore.indexes.json:23]: Yup. Deleted

File: lib/wallet/balance_chart.dart [hunk 3]
```diff
@@ -0,0 +1,328 @@
+import 'package:flutter/material.dart';
```
[the remark this answers is not in the document]
Comment (desimmons) [reply lib/wallet/balance_chart.dart:1]: Removed

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (robbie-daniels) [reply README.md:2]: Whoops you’re correct! I was mostly just thinking front the angle of: it seems unclear how people can use the client once they have it running

**Review by robbie-daniels — CHANGES_REQUESTED** [review]
* I believe we need to include copyright and license notices for our dependencies (I think most are MIT/BSD-3/ISC)

**Comment by desimmons** [comment]
> * I believe we need to include copyright and license notices for our dependencies (I think most are MIT/BSD-3/ISC)

I've pushed an update to include a `LicensePage` widget in the Drawer

**Review by robbie-daniels — APPROVED** [review]

**Review by robbie-daniels — APPROVED** [review]

**Pull request merged**